### DISCUSSION TITLE
GSoC 2023 - Pedagogical Interactive Notebooks for Algorithms Implemented in NetworkX

### DISCUSSION THREAD

**Discussion by PurviChaurasia** [networkx/networkx#6599] [Q&A]
Hi, I am Purvi Chaurasia. I have already contributed to the networkx repository in many ways and plan on continuing the same. As a part of GSoC 2023, I plan on submitting the following proposal for the project - Pedagogical Interactive Notebooks for Algorithms Implemented in NetworkX. @MridulS @rossbar Kindly provide feedback on the proposal and how I can further improve it. The proposal can be found [here](https://docs.google.com/document/d/16WOOjSmsXbT2yrgpQZpvWtfYqC1mmt66HLKH2FrZdOw/edit?usp=sharing). Waiting for your speedy response as I have to submit the proposal soon!

**Comment by MridulS** [comment]
BTW, please don't wait for our responses before submitting. The GSoC deadlines are pretty strict! Please go ahead and submit your proposals on the website ASAP.

**Reply by PurviChaurasia** [reply]
@MridulS any update on the feedback?

**Reply by PurviChaurasia** [reply]
@MridulS @rossbar I know you all are pretty swamped right now, but if you could just let me know if this proposal is good to go, it would be great. The submission deadline is approaching and I would like to incorporate any changes that you mention before it closes :)

**Reply by MridulS** [reply]
It looks good. Thanks!

**Reply by PurviChaurasia** [reply]
Thanks a lot :)

**Comment by PurviChaurasia** [comment]
Yep, I have submitted the proposal, however, would like to incorporate any changes that you suggest :)

**Comment by PurviChaurasia** [comment]
I also had a doubt, do you take one intern per project or one in total for GSoC? There isn't any information regarding this on the website.

**Reply by MridulS** [reply]
It's usually one per project (notebooks one is a special case where we can have multiple contributors), but it also depends on bunch of things like how many slots NumFOCUS gets + availability of mentor time.

**Reply by PurviChaurasia** [reply]
Alright thanks!
